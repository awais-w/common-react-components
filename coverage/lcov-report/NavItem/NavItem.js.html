<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for NavItem/NavItem.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../index.html">All files</a> / <a href="index.html">NavItem</a> NavItem.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>5/5</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>8/8</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>2/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>5/5</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">5x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react';
import { CSSTransition } from 'react-transition-group';
import PropTypes from 'prop-types';
import './NavItem.scss';
&nbsp;
const NavItem = ({ title, subTitle, summary, isActive, onClick, isComplete, currIndex }) =&gt; (
  &lt;div
    onClick={() =&gt; onClick({ desiredIndex: currIndex })}
    className={`step-title-container ${isActive ? 'active-item' : ''}`}&gt;
    &lt;div className='step-title-container__title'&gt;
      {isComplete ? (
        &lt;CSSTransition in timeout={300} classNames='step-title-container__complete' unmountOnExit&gt;
          &lt;div className='step-title-container__complete'&gt;â˜‘&lt;/div&gt;
        &lt;/CSSTransition&gt;
      ) : (
        ''
      )}
      {title}
    &lt;/div&gt;
&nbsp;
    {/* Only show description of step for active one */}
    {isActive &amp;&amp; &lt;div className='step-title-container__subtitle'&gt;{subTitle}&lt;/div&gt;}
&nbsp;
    {/* Show summaries on previous steps if we've got one */}
    {isComplete &amp;&amp; &lt;div className='step-title-container__summary'&gt;{summary}&lt;/div&gt;}
  &lt;/div&gt;
);
&nbsp;
NavItem.propTypes = {
  /**
   * Primary title for this step
   */
  title: PropTypes.string.isRequired,
  /**
   * Called when we click anywhere in nav item
   */
  onClick: PropTypes.func,
  /**
   * Description about whats required from the step
   */
  subTitle: PropTypes.string,
  /**
   * Show whats been completed in a previous step
   */
  summary: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
  /**
   * Whether we're currently displaying the content for this step
   */
  isActive: PropTypes.bool.isRequired,
  /**
   * Has this stage already been completed
   */
  isComplete: PropTypes.bool,
  /**
   * The current index we've reached upto
   */
  currIndex: PropTypes.number.isRequired,
};
&nbsp;
NavItem.defaultProps = {
  onClick: null,
  summary: '',
  subTitle: '',
  isComplete: true,
};
&nbsp;
export default NavItem;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Thu May 02 2019 15:57:09 GMT+0100 (British Summer Time)
</div>
</div>
<script src="../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../sorter.js"></script>
</body>
</html>
