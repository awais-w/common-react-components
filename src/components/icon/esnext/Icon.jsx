import React from 'react';
import PropTypes from 'prop-types';
import bem from '@argos/utils/esnext/bem';
import './Icon.scss';

const Icon = (props) => {
  const { isCustom, icon, onClick, id, disabled, color } = props;
  const cn = bem({ prefix: 'cmc-', block: isCustom ? 'icon-custom' : 'icon' });
  return (
    <i
      className={`${cn(null, color, { disabled, [icon]: isCustom, clickable: onClick })}`}
      id={id || undefined}
      role={onClick ? 'button' : 'icon'}
      onClick={onClick}>
      {icon}
    </i>
  );
};

Icon.propTypes = {
  icon: PropTypes.string.isRequired,
  isCustom: PropTypes.bool,
  onClick: PropTypes.func,
  id: PropTypes.string,
  disabled: PropTypes.bool,
  color: PropTypes.oneOf([undefined, 'white', 'red', 'yellow', 'green', 'blue', 'black', 'grey']),
};

Icon.defaultProps = {
  onClick: null,
  isCustom: false,
};

export default Icon;
