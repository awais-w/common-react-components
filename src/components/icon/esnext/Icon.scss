@import '@cmc/common-styles/src/scss/_variables.scss';

@font-face {
  font-family: 'Material Icons';
  font-style: normal;
  font-weight: 400;
  src: local('Material Icons'), local('MaterialIcons-Regular'),
    url(https://fonts.gstatic.com/s/materialicons/v42/flUhRq6tzZclQEJ-Vdg-IuiaDsNcIhQ8tQ.woff2) format('woff2');
}

@font-face {
  font-family: 'icomoon';
  src: url('fonts/icomoon.woff?y5vhvj') format('woff');
  font-style: normal;
  font-weight: 400;
}

.cmc-icon {
  font-family: 'Material Icons';
  font-weight: normal;
  font-style: normal;
  font-size: $icon-size-standard;
  line-height: 1;
  letter-spacing: normal;
  text-transform: none;
  display: inline-block;
  white-space: nowrap;
  word-wrap: normal;
  direction: ltr;
  font-feature-settings: 'liga';
  -webkit-font-smoothing: subpixel-antialiased;
}

.cmc-icon-custom {
  /* use !important to prevent issues with browser extensions that change fonts */
  font-family: 'icomoon' !important;
  font-style: normal;
  font-weight: normal;
  font-variant: normal;
  text-transform: none;
  line-height: 1;
  font-size: 0 !important;

  /* Better Font Rendering =========== */
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  &:before {
    font-size: $font-size-large;
  }
}

i.cmc-icon,
i.cmc-icon-custom {
  &.clickable {
    cursor: pointer;
  }
  &--white {
    color: $color-white;
  }
  &--red {
    color: $color-red;
  }
  &--yellow {
    color: $color-star-stroke;
  }
  &--green {
    color: $color-green;
  }
  &--blue {
    color: $primary-color;
  }
  &--black {
    color: $color-black;
  }
  &--grey {
    color: $color-grey-dark;
  }
}

/* Custom icons */
.icon-change-adress:before {
  content: '\e900';
}
.icon-refund-delivery-charge:before {
  content: '\e914';
}
.icon-leader:before {
  content: '\e912';
}
.icon-url:before {
  content: '\e913';
}
.icon-currency:before {
  content: '\e90f';
}
.icon-to-left:before {
  content: '\e910';
}
.icon-to-right:before {
  content: '\e911';
}
.icon-address-lookup:before {
  content: '\e90e';
}
.icon-returns:before {
  content: '\e90d';
}
.icon-collected:before {
  content: '\e901';
}
.icon-created:before {
  content: '\e902';
}
.icon-delivered:before {
  content: '\e903';
}
.icon-in-progress:before {
  content: '\e904';
}
.icon-in-transit:before {
  content: '\e905';
}
.icon-ready-to-collect:before {
  content: '\e906';
}
.icon-refunded:before {
  content: '\e907';
}
.icon-return-failed:before {
  content: '\e908';
}
.icon-return-in-progress:before {
  content: '\e909';
}
.icon-returned:before {
  content: '\e90a';
}
.icon-rebook-delivery:before {
  content: '\e90b';
}
.icon-price-history:before {
  content: '\e90c';
}
.icon-pointer:before {
  content: '\e915';
}
.icon-contact-history:before {
  content: '\e916';
}
.icon-order-history:before {
  content: '\e917';
}
.icon-failed-payment:before {
  content: '\e918';
}
.icon-sticky-note:before {
  content: '\e919';
}
