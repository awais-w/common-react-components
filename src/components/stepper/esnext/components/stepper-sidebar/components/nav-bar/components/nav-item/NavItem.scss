@import '@cmc/common-styles/src/scss/_variables.scss';

.cmc-nav-item {
  @include font-standard;
  border-bottom: 1px solid $color-grey-mid;
  padding: $double;
  cursor: pointer;

  &.locked {
    cursor: not-allowed;
  }

  &--active {
    background-color: $color-grey-light;
  }
  &__title {
    @include font-bold;
    text-transform: uppercase;
    font-size: $font-size-large;
    position: relative;
    display: flex;
    align-items: center;
    padding-right: $super-double;
  }
  &__subtitle {
    padding-top: $base;
    font-size: $font-size-medium;
  }
  &__summary {
    padding-top: $base;
  }
  &__complete {
    color: $color-green;
    transform: scale(1.25);
    position: absolute;
    right: 0;
    top: 0;
  }

  // Animation of "complete" icon
  &__complete-enter {
    opacity: 0.01;
    transform: translateY(-100%) scale(0.75);
  }
  &__complete-enter-active {
    opacity: 1;
    transform: translateY(0%) scale(1.25);
    transition: all 300ms ease-out;
  }
  &__complete-exit {
    opacity: 1;
    transform: scale(1.25);
  }
  &__complete-exit-active {
    opacity: 0;
    transform: scale(4);
    transition: all 300ms ease-in;
  }

  .cmc-icon,
  .cmc-icon-custom {
    margin-right: $half;
    transition: color 0.5s;
  }
}

.cmc-stepper--nav-minimize {
  .cmc-nav-item {
    display: flex;
    justify-content: center;
    align-items: center;
    flex-wrap: wrap;
    padding: $base 0;
    &__title {
      padding-right: 0;
    }
    .cmc-icon {
      margin: 0;
      font-size: $font-size-h1;
      cursor: pointer;
    }
    .cmc-icon-custom {
      margin: 0;
      cursor: pointer;
      &::before {
        font-size: $font-size-h1;
      }
    }
    &.locked {
      cursor: pointer;
    }
    &__complete {
      & + .cmc-icon,
      & + .cmc-icon-custom {
        color: $color-green;
      }
    }
    &__summary {
      @include font-condensed-extra-bold;
      padding-top: $half;
      width: 100%;
      text-align: center;
      font-size: $font-size-small;
      line-height: 1;
    }
  }
}
