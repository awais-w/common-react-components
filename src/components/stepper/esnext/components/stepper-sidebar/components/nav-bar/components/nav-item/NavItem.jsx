import React from 'react';
import { CSSTransition } from 'react-transition-group';
import PropTypes from 'prop-types';
import Icon from '@cmc/icon/esnext/Icon';
import bem from '@argos/utils/esnext/bem';
import classNames from 'classnames';
import './NavItem.scss';
import NavItemConstants from './constants';

const cn = bem({ prefix: 'cmc-', block: 'nav-item' });

const NavItem = ({
  icon,
  isCustomIcon,
  title,
  subTitle,
  summary,
  shortSummary,
  isActive,
  handleClick,
  isComplete,
  currIndex,
  summaryStyle,
  allowNavigation,
  minimizeNav,
}) => {
  const classes = classNames(cn(), {
    [cn(null, 'active')]: isActive,
    locked: !allowNavigation,
  });

  return (
    <div onClick={() => handleClick({ desiredIndex: currIndex })} className={classes}>
      <div className={cn('title')}>
        {isComplete ? (
          <CSSTransition in timeout={300} classNames={cn('complete')} unmountOnExit>
            <div className={cn('complete')}>{!minimizeNav && <Icon icon={'check_circle'} />}</div>
          </CSSTransition>
        ) : (
          ''
        )}
        {icon && <Icon icon={icon} isCustom={isCustomIcon} />}
        {!minimizeNav && title}
      </div>

      {/* Only show description of step for active one */}
      {isActive && !minimizeNav && <div className={cn('subtitle')}>{subTitle}</div>}

      {/* Show summaries on previous steps if we've got one */}
      {summary &&
        ((isComplete && summaryStyle === 'after-completion') || (isActive && summaryStyle === 'real-time')) && (
          <div className={cn('summary')}>{minimizeNav ? shortSummary : summary}</div>
        )}
    </div>
  );
};
NavItem.propTypes = {
  /**
   * Icon for this step
   */
  icon: PropTypes.string.isRequired,
  /**
   * Primary title for this step
   */
  title: PropTypes.string.isRequired,
  /**
   * Called when we click anywhere in nav item
   */
  onClick: PropTypes.func,
  /**
   * Description about whats required from the step
   */
  subTitle: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
  /**
   * Show whats been completed in a previous step
   */
  summary: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
  /**
   * Show summary in minimzed state
   */
  shortSummary: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
  /**
   * Whether we're currently displaying the content for this step
   */
  isActive: PropTypes.bool.isRequired,
  /**
   * Has this stage already been completed
   */
  isComplete: PropTypes.bool,
  /**
   * Is Nav minimized
   */
  minimizeNav: PropTypes.bool,
  /**
   * The current index we've reached upto
   */
  currIndex: PropTypes.number.isRequired,
  /**
   * It will display the sumarry of the step as follow:
   * - real-time: always visible
   * - after-completion: only when the step is completed
   * - hide: never visible
   */
  summaryStyle: PropTypes.oneOf(NavItemConstants.summaryStyles),
  /**
   * Determines when the stepper should allow to move to a previous step.
   */
  allowNavigation: PropTypes.bool,
};

NavItem.defaultProps = {
  icon: '',
  summary: '',
  shortSummary: '',
  subTitle: '',
  isComplete: true,
  summaryStyle: 'after-completion',
};

export default NavItem;
