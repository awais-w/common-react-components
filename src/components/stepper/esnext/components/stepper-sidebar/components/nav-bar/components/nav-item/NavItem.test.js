import React from 'react';
import { shallow } from 'enzyme';
import Icon from '@cmc/icon/esnext/Icon';
import NavItem from './NavItem';

const defaultProps = {
  handleClick: jest.fn(),
  icon: 'favorite',
  title: 'My Title',
  summary: 'My Summary',
  shortSummary: '£XX.xx',
  isActive: false,
  isComplete: false,
  minimizeNav: false,
  currIndex: 0,
};

const setup = ({ props }) => {
  const component = shallow(<NavItem {...props} />);
  return {
    wrapper: component,
    getStepTitleContainer: () => component.find('.cmc-nav-item__title'),
    getCompleteNavItem: () => component.find('.cmc-nav-item__complete'),
    getStepSummary: () => component.find('.cmc-nav-item__summary'),
    getStepSubtitle: () => component.find('.cmc-nav-item__subtitle'),
    getNavItemContainer: () => component.find('.cmc-nav-item'),
  };
};

describe('NavItem', () => {
  const component = setup({ props: defaultProps });

  it('should match default snapshot', () => {
    expect(component.wrapper).toMatchSnapshot();
  });

  it('Always renders a title with the correct icon and text', () => {
    component.wrapper.setProps({ minimizeNav: true });
    expect(component.wrapper.find(Icon)).toExist();
    component.wrapper.setProps({ minimizeNav: false });
    expect(component.getStepTitleContainer().text()).toBe(`<Icon />${defaultProps.title}`);
  });

  describe('prop : isComplete', () => {
    it('Renders a ticked checkbox when true', () => {
      expect(component.getCompleteNavItem().exists()).toBe(false);
      component.wrapper.setProps({ isComplete: true });
      expect(component.getCompleteNavItem().exists()).toBe(true);
    });

    it('Renders a summary/shortSummary with the correct text when true', () => {
      component.wrapper.setProps({ isComplete: false });
      expect(component.getStepSummary().exists()).toBe(false);
      component.wrapper.setProps({ minimizeNav: true, isComplete: true, shortSummary: '£00.00' });
      let summaryNode = component.getStepSummary();
      expect(summaryNode.text()).toBe('£00.00');
      component.wrapper.setProps({ minimizeNav: false, summary: '...' });
      summaryNode = component.getStepSummary();
      expect(summaryNode.text()).toBe('...');
    });
  });

  describe('prop : isActive', () => {
    it('Renders a subTitle with the correct text when true', () => {
      expect(component.getStepSubtitle().exists()).toBe(false);
      component.wrapper.setProps({ isActive: true, subTitle: '...' });
      const summaryNode = component.getStepSubtitle();
      expect(summaryNode.text()).toBe('...');
    });
  });

  describe('prop : onClick', () => {
    it('Calls handler when clicked', () => {
      component.getNavItemContainer().simulate('click');
      expect(defaultProps.handleClick).toBeCalled();
    });
  });

  describe('prop : summaryStyle', () => {
    describe('after-completion', () => {
      it('Renders summary with correct text when isComplete', () => {
        component.wrapper.setProps({ isActive: true, summaryStyle: 'after-completion', isComplete: true });
        expect(component.getStepSummary().text()).toBe('...');
      });
      it('Does not render summary when NOT isComplete', () => {
        component.wrapper.setProps({ isActive: true, summaryStyle: 'after-completion', isComplete: false });
        expect(component.getStepSummary().exists()).toBe(false);
      });
    });
    describe('hide', () => {
      it('Does not render summary when isComplete', () => {
        component.wrapper.setProps({ isActive: true, summaryStyle: 'hide', isComplete: true });
        expect(component.getStepSummary().exists()).toBe(false);
      });
      it('Does not render summary when NOT isComplete', () => {
        component.wrapper.setProps({ isActive: true, summaryStyle: 'hide', isComplete: false });
        expect(component.getStepSummary().exists()).toBe(false);
      });
    });
    describe('real-time', () => {
      it('Renders summary with correct text when isComplete', () => {
        component.wrapper.setProps({ isActive: true, summaryStyle: 'real-time', isComplete: true });
        expect(component.getStepSummary().text()).toBe('...');
      });
      it('Renders summary with correct text when NOT isComplete', () => {
        component.wrapper.setProps({ isActive: true, summaryStyle: 'real-time', isComplete: false });
        expect(component.getStepSummary().text()).toBe('...');
      });
    });
  });
});
