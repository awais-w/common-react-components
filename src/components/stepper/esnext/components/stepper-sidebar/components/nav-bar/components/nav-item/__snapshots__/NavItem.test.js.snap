// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavItem should match default snapshot 1`] = `
<div
  className="cmc-nav-item locked"
  onClick={[Function]}
>
  <div
    className="cmc-nav-item__title"
  >
    <Icon
      icon="favorite"
      isCustom={false}
      onClick={null}
    />
    My Title
  </div>
</div>
`;
